hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssCryptographyManager.java

```diff
@@ -129,6 +139,14 @@ private CompletableFuture<LedgerIdWithSignature> computeAndSignLedgerIdIfApplica
                     final var signature = gossip.sign(ledgerId.publicKey().toBytes());
 
                     final BitSet tssVoteBitSet = computeTssVoteBitSet(validTssOps);
+
+                    // calculate the time it took to aggregate and compute ledger id, public shares and private shares
```

Comment:
The private shares are not being computed here, we can drop it from the comment.   Measuring the other two aggregation operations are enough to give a feel for how expensive aggregation is.

Reply:
Good catch, a bit misleading! The comment is updated, ty!